aws_kube_params.inc

```diff
@@ -18,3 +18,19 @@ export KUBEMARK_AUTOSCALER_MAX_NODES=${KUBEMARK_NUM_NODES}
 export NUM_REPLICAS=${KUBEMARK_NUM_NODES}
 export NUM_WINDOWS_NODES=0
 export AWS_SSH_KEY=$HOME/.ssh/id_rsa
+
+export PRESET_INSTANCES_ENABLED=yes
+export PRESET_KUBE_MASTER_IP=3.134.111.193
```

Comment:
Use static IP as MASTER_IP and try to use Internal_IP for all nodes

Reply:
Done. Will show in next iteration.